docker/README.md

```diff
@@ -0,0 +1,104 @@
+Miaou development environment using Docker
+===
+**Build a local tiny Virtual Machine which allows you to run Miaou on any OS (including Windows/OS X)**
+
+
+## Requirements
+
+ - [Docker-machine](https://docs.docker.com/machine/#installation) ([version 0.3.0+](https://github.com/docker/machine/releases)) => an executable to rename `docker-machine` and to move somewhere in your `PATH`
+ - [VirtualBox](https://www.virtualbox.org/wiki/Downloads)
+
+
+## What's inside?
+
+### docker-machine-dev(.bat/.sh)
+A simple yet powerful script that use Docker Machine to build a local Docker aware VM (it uses boot2docker in the background). 
+It's also used to set some useful aliases in the VM (such as`docker-compose` which run Docker Compose in a container).
+
+### Dockerfile
+ - Miscellaneous useful Ubuntu packages
+ - Python (needed to build some `npm` dependencies)
+ - Ruby and `sass`
+ - IOJS with `nodemon`, `uglify-js` and `buster` globally installed
+
+### docker-entrypoint.sh
+This file is the default ENTRYPOINT of the image. It will be executed each time you run the Miaou container. This is where you should put all your project specific stuffs (e.g. run services, run a Makefile, etc.).
+
+### docker-compose.yml
+This file (using Docker Compose) is the recommanded way to build the whole environment. It will run and link several containers through a single command. 
+Keep in mind that this could also be done using many long `docker` commands without the need of `docker-compose` at all.
+
+
+## How to use?
+
+### First time (assuming you already cloned your fork)
+
+ 1. Go to the docker directory `cd /path/to/miaou/docker` and type `docker-machine-dev` (if on Windows) or `./docker-machine-dev.sh` (else).
+If you followed the [Requirements](#requirements), after a few seconds you will be landed in the VM where you will be able to run `docker` commands.
+
+ 2. Now that you are in your VM, you can build the image `docker-compose` with the alias: 
+	```
+	install-docker-compose
+	```
+ 
+ 3. Then, again, `cd` to the docker directory (the VM shared your `C:\Users\` (windows) or `/Users/` (os x) folder as `/c/Users/` or `/Users/` --- see [complete reference here](https://github.com/boot2docker/boot2docker#virtualbox-guest-additions)).
+ 
+ 4. Build and run the whole Miaou environment with:
+	```
+	docker-compose run --service-ports miaou
+	```
+It will leave you in the Miaou container at `/var/www/html` where you will be able to manage everything like if you were on a server with everything installed.
+If you `exit` this container, it will be stopped then you'll have 2 solutions to come back in it:
+    - Remove the container with `docker rm docker_miaou_run_1` then run it again with `docker-compose run --service-ports miaou` 
+    - Restart the container with `docker start docker_miaou_run_1` then execute `bash` in it with `docker exec -it docker_miaou_run_1 bash`
```

Comment:
`docker-compose up -d` doesn't fit well here because our main container default command (`bash`) is not a long running process, so the container just start and call `bash`... then exit...

`docker-compose run miaou` then creates another "miaou container"... that's why, I recommend to always "manually" remove/recreate or restart an existing "miaou container"